autogen.sh

```diff
@@ -1,53 +0,0 @@
-#!/bin/sh
-
-warn() {
-	echo "WARNING: $@" 1>&2
-}
-
-case `uname -s` in
-Darwin)
-	LIBTOOLIZE=glibtoolize
-	;;
-FreeBSD)
-	LIBTOOLIZE=libtoolize
-	;;
-Linux)
-	LIBTOOLIZE=libtoolize
-	;;
-SunOS)
-	LIBTOOLIZE=libtoolize
-	;;
-*)
-	warn "unrecognized platform:" `uname -s`
-	LIBTOOLIZE=libtoolize
-esac
-
-automake_version=`automake --version | tr ' ' '\n' | egrep '^[0-9]\.[0-9a-z.-]+'`
-if [ -z "$automake_version" ] ; then
-	warn "unable to determine automake version"
-else
-	case $automake_version in
-		0.*|1.[0-8]|1.[0-8][.-]*)
-			warn "automake ($automake_version) detected; 1.9 or newer recommended"
-			;;
-		*)
-			;;
-	esac
-fi
-
-# check for varnishapi.m4 in custom paths
-dataroot=$(pkg-config --variable=datarootdir varnishapi 2>/dev/null)
-if [ -z "$dataroot" ] ; then
-	cat >&2 <<'EOF'
-Package varnishapi was not found in the pkg-config search path.
-Perhaps you should add the directory containing `varnishapi.pc'
-to the PKG_CONFIG_PATH environment variable
-EOF
-	exit 1
-fi
-set -ex
-aclocal -I m4 -I ${dataroot}/aclocal
```

Comment:
... and this, otherwise I don't see how a custom varnish install location would work
```
slink@haggis:~/Devel/varnish-git/libvmod-example (dridi_varnish-5)$ export PKG_CONFIG_PATH=$PREFIX/lib/pkgconfig
slink@haggis:~/Devel/varnish-git/libvmod-example (dridi_varnish-5)$ ./bootstrap 
libtoolize: putting auxiliary files in AC_CONFIG_AUX_DIR, `build-aux'.
libtoolize: copying file `build-aux/ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIR, `m4'.
libtoolize: copying file `m4/libtool.m4'
libtoolize: copying file `m4/ltoptions.m4'
libtoolize: copying file `m4/ltsugar.m4'
libtoolize: copying file `m4/ltversion.m4'
libtoolize: copying file `m4/lt~obsolete.m4'
src/Makefile.am:4: error: 'vmod_LTLIBRARIES' is used but 'vmoddir' is undefined
automake: warnings are treated as errors
src/Makefile.am:6: warning: variable 'libvmod_example_la_SOURCES' is defined but no program or
src/Makefile.am:6: library has 'libvmod_example_la' as canonical name (possible typo)
src/Makefile.am:9: warning: variable 'nodist_libvmod_example_la_SOURCES' is defined but no program or
src/Makefile.am:9: library has 'libvmod_example_la' as canonical name (possible typo)

```